readme.md

```diff
@@ -76,19 +76,19 @@ PR welcome for additional commonly needed types and docs improvements. Read the
 npm install type-fest
 ```
 
-*Requires TypeScript >=5.1*
+_Requires TypeScript >=5.1_
```

Comment:
Don't make unrelated changes.

Reply:
My bad, forgot to turn Prettier off and didn't notice all the changes it made. All fixed now

Reply:
It's not fixed.

Reply:
Well, this was awkward. Now I don't see much in the readme diff. Sorry about that.